app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,330 @@
+@import "colors";
+@import "foundation_and_overrides";
+@import url(https://fonts.googleapis.com/css?family=Mako);
+@import url(https://fonts.googleapis.com/css?family=Lora);
+
+
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ *= require foundation_and_overrides
+
+ */
+
+
+ // @mixin vertical-align {
+ //   position: relative;
+ //   top: 50%;
+ //   -webkit-transform: translateY(-50%);
+ //   -ms-transform: translateY(-50%);
+ //   transform: translateY(-50%);
+ // }
+
+
+ .pad-vert {
+    padding-top: 50px;
+    padding-bottom: 50px;
+}
+
+.box-background {
+  background-color: $pastel-grey;
+}
+
+// .logo-right{
+//   display: inline-block;
+//   vertical-align: middle;
+//   width: 50%;
+//  //  @include vertical-align;
+// }
+//
+// .time-butt{
+//   display: inline-block;
+//   vertical-align: middle;
+//   width: 50%;
+//   // @include vertical-align;
+// }
+ .top {
+   background: url("https://dl.dropboxusercontent.com/u/18396109/gray_background.jpg") no-repeat center center;
```

Comment:
URL `https://dl.dropboxusercontent.com/u/18396109/gray_background.jpg` should not contain protocol or domain